node/src/actors/chain_manager/handlers.rs

```diff
@@ -1309,7 +1309,7 @@ impl Handler<GetReputation> for ChainManager {
         };
 
         let identities = if all {
-            rep_eng.trs().identities().map(|(k, _v)| k).collect()
+            rep_eng.ars().active_identities().collect()
```

Comment:
Will this make the list ordered? If not, we can use `rep_eng.get_rep_ordered_ars_list()`

Reply:
agree